.github/workflows/traefik-secure-test.yml

```diff
@@ -0,0 +1,197 @@
+        run: |
+          docker compose -f compose.traefik.secure.yml config > /dev/null
+          echo "✅ compose.traefik.secure.yml syntax is valid"
+
+  test-secure-setup:
+    name: Test Traefik Secure Setup
+    runs-on: ubuntu-latest
+    needs: validate-compose
+    permissions:
+      contents: read
+      actions: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8  # v6.0.1
+
+      - name: Free up disk space
+        run: |
+          echo "=== Disk space before cleanup ==="
+          df -h
+          echo "=== Cleaning up ==="
+          docker system prune -af --volumes || true
+          sudo rm -rf /usr/local/lib/android || true
+          sudo rm -rf /usr/share/dotnet || true
+          sudo rm -rf /opt/ghc || true
+          sudo rm -rf /opt/hostedtoolcache || true
+          echo "=== Disk space after cleanup ==="
+          df -h
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@e468171a9de216ec08956ac3ada2f0791b6bd435  # v3.11.1
+
+      - name: Create test .env file
+        run: |
+          cp ./dtaas/.env.example .env
+          # Replace placeholders with test values
+          sed -i 's|OAUTH_URL=.*|OAUTH_URL=https://gitlab.com|g' .env
+          sed -i 's|OAUTH_CLIENT_ID=.*|OAUTH_CLIENT_ID=test_client_id|g' .env
+          sed -i 's|OAUTH_CLIENT_SECRET=.*|OAUTH_CLIENT_SECRET=test_secret|g' .env
+          sed -i 's|OAUTH_SECRET=.*|OAUTH_SECRET=test_oauth_secret_key_12345|g' .env
+          sed -i 's|username1=.*|username1=user1|g' .env
+          sed -i 's|username2=.*|username2=user2|g' .env
+
+      - name: Build workspace-nouveau image
+        uses: docker/build-push-action@263435318d21b8e681c14492fe198d362a7d2c83  # v6.18.0
+        with:
+          context: .
+          file: ./Dockerfile
+          push: false
+          tags: workspace-nouveau:latest
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+      - name: Pull required images
+        run: |
+          docker pull traefik:v3.2
+          docker pull thomseddon/traefik-forward-auth:latest
+          docker pull ghcr.io/into-cps-association/dtaas-web:latest || \
+            echo "⚠️ DTaaS web image not available, will skip"
+          docker pull mltooling/ml-workspace-minimal:0.13.2
+
+      - name: Start services (excluding client if unavailable)
+        run: |
+          # Try to start all services
+          if docker compose -f compose.traefik.secure.yml up -d 2>&1 | grep -q "pull access denied\|not found"; then
+            echo "⚠️ Client image not available, starting without client service"
+            docker compose -f compose.traefik.secure.yml up -d traefik traefik-forward-auth user1 user2
+          else
+            docker compose -f compose.traefik.secure.yml up -d
+          fi
+          echo "⏳ Waiting for services to be ready..."
+          sleep 30
+
+      - name: Check service status
```

Comment:
Use [this coding pattern](https://github.com/INTO-CPS-Association/workspace/blob/main/.github/workflows/traefik-test.yml#L64)